schema/data/dev/ciip_user_organisation.sql

```diff
@@ -14,7 +14,8 @@ begin;
   overriding system value
   values
   (1, 3, 8, 'approved'),
-  (2, 3, 7, 'approved')
+  (2, 3, 7, 'approved'),
+  (3, 6, 7, 'approved')
```

Comment:
will discuss with @matthieu-foucault on how to best set up db for cypress when he is back in the office (and not before!)